Gemfile

```diff
@@ -11,10 +11,8 @@ def local_gem(gem_name, dir: gem_name, **opts)
 end
 
 group :development do
-  local_gem 'activerecord-honeycomb'
   local_gem 'rack-honeycomb'
   local_gem 'faraday-honeycomb'
-  local_gem 'sequel-honeycomb'
```

Comment:
@jmhodges-mode - removing these dependencies and running `bundle` doesn't actually update the `Gemfile.lock` file - it looks like the function above ([click for reference](https://github.com/mode/beeline-ruby/blob/c1b29e408764b1decf78cd03d2e712ef5e7befa5/Gemfile#L5)) doesn't account for removing gems.

I'd like to leave this as-is since they don't seem to be large dependencies and it doesn't impact what we're doing with this PR. Thoughts?

Reply:
Yeah bring these back and leave em alone

Reply:
Updated.